tests/test_registration.py

```diff
@@ -1,7 +1,68 @@
 import pytest
+from app.src.app import app
 from app.src.auth import register
 from app.src.models import User, db
 
-from flask import Flask, request, redirect, url_for
+@pytest.fixture
+def client():
+    app.config['TESTING'] = True
+    with app.test_client() as client:
+        yield client
 
-# TODO
\ No newline at end of file
+def test_register(client):
+    response1 = client.post('/register', data={
```

Comment:
I think some general comments explaining each client request could be helpful just for clarification